src/pages/Hwisung/Login/Login.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+// import Btn from './Components/Btn/Btn';
+// import IdPw from './Components/IdPw/IdPw';
+import { withRouter } from 'react-router-dom';
+import './Login.scss';
+
+class LoginHwisung extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      InputId: '',
+      InputPw: '',
+      BtnOnOff: 'LoginBtn',
+    };
+  }
+  goToMain = () => {
+    this.props.history.push('/Main'); //컴포넌트가 아니라 경로
+  };
+
+  handleIdInput = event => {
+    this.setState({
+      InputId: `${event.target.value}`,
```

Comment:
```suggestion
      InputId: event.target.value
```
그냥 이렇게 적어주셔도 될 것 같아요! 🙂